### ISSUE TITLE
Helm 3: Failed to install chart in a namespace with a release name already used in a different namespace

### ISSUE THREAD

**Issue by hickeyma** [helm/helm#5857]
When I try to install chart with a release name unique to a namespace, it errors as follows:

```console
$ h3 upgrade --install nginx-ingress stable/nginx-ingress
Release "nginx-ingress" does not exist. Installing it now.
Error: release nginx-ingress failed: clusterroles.rbac.authorization.k8s.io "nginx-ingress" already exists
```

Environment is setup as follows with release `nginx-ingress` already deployed to namespace `nginx-ingress`:
```console
$ h3 ls
NAME         	NAMESPACE    	REVISION	UPDATED                                	STATUS  	CHART               
nginx-ingress	nginx-ingress	1       	2019-06-07 16:57:52.564963291 +0100 IST	deployed	nginx-ingress-1.6.16

$ h3 ls
NAME      	NAMESPACE	REVISION	UPDATED                                	STATUS  	CHART           
memcached 	myapp1   	1       	2019-06-07 16:59:19.597534923 +0100 IST	deployed	memcached-2.8.2 
postgresql	myapp1   	1       	2019-06-07 16:58:52.354528705 +0100 IST	deployed	postgresql-5.3.4

$ h3 upgrade --install nginx-ingress stable/nginx-ingress
Release "nginx-ingress" does not exist. Installing it now.
Error: release nginx-ingress failed: clusterroles.rbac.authorization.k8s.io "nginx-ingress" already exists
$ h3 ls
NAME         	NAMESPACE	REVISION	UPDATED                                	STATUS  	CHART               
memcached    	myapp1   	1       	2019-06-07 16:59:19.597534923 +0100 IST	deployed	memcached-2.8.2     
nginx-ingress	myapp1   	1       	2019-06-07 16:59:45.837896488 +0100 IST	failed  	nginx-ingress-1.6.16
postgresql   	myapp1   	1       	2019-06-07 16:58:52.354528705 +0100 IST	deployed	postgresql-5.3.4    
```

Output of `helm version`:
```console
version.BuildInfo{Version:"v3.0.0-alpha.1", GitCommit:"b9a54967f838723fe241172a6b94d18caf8bcdca", GitTreeState:"clean"}
```

Output of `kubectl version`:
```console
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.6", GitCommit:"abdda3f9fefa29172298a2e42f5102e777a8ec25", GitTreeState:"clean", BuildDate:"2019-05-08T13:53:53Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.0", GitCommit:"ddf47ac13c1a9483ea035a79cd7c10005ff21a6d", GitTreeState:"clean", BuildDate:"2018-12-03T20:56:12Z", GoVersion:"go1.11.2", Compiler:"gc", Platform:"linux/amd64"}
```

Cloud Provider/Platform (AKS, GKE, Minikube etc.): 
kubeadm-dind-cluster

**Comment by bacongobbler** [comment]
I believe this is the same design as it is in Helm 2, right? If a resource has already been created by an external entity (either another `helm install` or a `kubectl create`), `helm install` will not "adopt" that resource and will fail instead. However, the release is still created, but ends in the `FAILED` state

I believe this may be a similar report of #3319.

**Comment by hickeyma** [comment]
It probably makes most sense to follow the same practice as per Helm 2 by setting the release to `FAILED` state when resources are already created.

If this is undocumented, we might need to document about already created resources especially now that we are allowing charts to be installed with the same release name in different namespaces. Users will now probably get caught out by charts where they assume the resources are deployed at namespace scope  but may also have resources deployed  at cluster scoped, which they are unaware of.  The now probably increases the possibility of conflicts! :)

**Comment by bacongobbler** [comment]
fixed in #6542
